### PR TITLE
chore(eslint): upgrade to eslint-config-airbnb@18

### PR DESCRIPTION
**Pull request by gautier-lefebvre**
https://github.com/airbnb/javascript changes a rule via https://github.com/airbnb/javascript/pull/1863, making it mandatory to use parenthesis in arrow function even if there is only one parameter.

I personnally don't care much about this rule, but let me know what you think, should we use the old rule or the new rule?

### CODE REVIEW

File: src/renderer/react/app/nodes/editNodeModal/EditNodeModal.jsx [hunk 1]
```diff
@@ -16,14 +16,14 @@ import { formShape } from 'react/shapes/form';
...
+  = (props) => mapValues(props.node || {}, (value) => createFormField({ value }));
 
 /**
  * Make a list of available parents.
  */
 const getOptionsSelector = createSelector(
-  props => props.node,
-  props => props.nodeList,
+  (props) => props.node,
+  (props) => props.nodeList,
```
Comment (sylvainar) [inline src/renderer/react/app/nodes/editNodeModal/EditNodeModal.jsx]: meh

File: src/renderer/react/app/nodes/NodeManager.jsx [hunk 2]
```diff
@@ -13,13 +13,13 @@ import PromptNewNodeModal from './editNodeModal/EditNodeModal';
...
-    state => ({
+    (state) => ({
       nodeList: state.nodes,
     }),
-    dispatch => ({
-      onCreateNode: node => dispatch(create(node)),
-      onUpdateNode: node => dispatch(update(node)),
-      onRemoveNode: node => dispatch(remove(node)),
+    (dispatch) => ({
+      onCreateNode: (node) => dispatch(create(node)),
```
Comment (sylvainar) [inline src/renderer/react/app/nodes/NodeManager.jsx]: meh

File: src/renderer/react/app/nodes/nodeList/NodeManagerList.jsx [hunk 3]
```diff
@@ -86,10 +86,10 @@ export default class NodeManagerList extends React.PureComponent {
...
    */
   renderTree = createSelector(
-    props => props.nodeList,
-    props => props.onAddNode,
-    props => props.onEditNode,
-    props => props.onDeleteNode,
+    (props) => props.nodeList,
+    (props) => props.onAddNode,
+    (props) => props.onEditNode,
+    (props) => props.onDeleteNode,
```
Comment (sylvainar) [inline src/renderer/react/app/nodes/nodeList/NodeManagerList.jsx]: meh again

File: src/renderer/redux/middlewares/persistState.js [hunk 4]
```diff
@@ -8,7 +8,7 @@ import { saveState } from 'services/state';
  * const store = createStore(reducers, applyMiddleware(middleware));
  * @returns {object} The calculated state.
  */
-export default () => store => next => (action) => {
+export default () => (store) => (next) => (action) => {
```
Comment (sylvainar) [inline src/renderer/redux/middlewares/persistState.js]: double meh

**Review by sylvainar — COMMENTED** [review]
I've put some "meh" when it bugged me the most.

However i'm neither in favor nor opposed to it, what's the current trend in opensource projects?

**Comment by gautier-lefebvre** [comment]
Don't know, I understand the point but sometimes it feels really useless, I would prefer to disable it

File: src/renderer/redux/middlewares/persistState.js (hunk 4 above)
Comment (dblervaque) [reply src/renderer/redux/middlewares/persistState.js]: MONSTER MEH !

**Review by dblervaque — APPROVED** [review]

**Pull request merged**